### PR TITLE
feature(docs): added pr and issue templates

### PR DESCRIPTION
**Pull request by Ballena0**
This pr is focused on discussing the usage and creation of this templates.
PR enfocado en la creación y modificación de estos templates.

### CODE REVIEW

File: docs/issue_template.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+*borra esto en tu pr*
+
+
+## Type of change/ tipo de cambio:
+
+- [ ] Bug fix (corrección de algún error)
+- [ ] New feature (nueva funcionalidad)
+- [ ] Breaking change (cambio que puede romper la aplicación)
+- [ ] Docs (documentación)
+
```
Comment (Javendanon) [inline docs/issue_template.md:19]: Here is missing something like technical debt 🤔 what do you think?

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+*borra esto en tu pr*
+
+
+## Type of change/ tipo de cambio:
+
+- [ ] Bug fix (corrección de algún error)
+- [ ] New feature (nueva funcionalidad)
+- [ ] Breaking change (cambio que puede romper la aplicación)
+- [ ] Docs (documentación)
+
```
Comment (Javendanon) [inline docs/pull_request_template.md:16]: same here

**Review by Javendanon — APPROVED** [review]
LGTM, minor suggestion 👍

**Comment by Javendanon** [comment]
FTR: I forgot to mention it but i created a new branch `dev` to target all the pull request to that branch

**Comment by Ballena0** [comment]
> FTR: I forgot to mention it but i created a new branch `dev` to target all the pull request to that branch

OMW to redirect

File: docs/issue_template.md (hunk 1 above)
Comment (Ballena0) [reply docs/issue_template.md:19]: sounds good, i'll add it

**Pull request merged**